sdk/src/main/scala/io/horizen/account/api/http/route/AccountTransactionApiRoute.scala

```diff
@@ -537,6 +538,58 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
     }
   }
 
+  def pagedForgersStakesByForger: Route = (post & path("pagedForgersStakesByForger")) {
+    withBasicAuth {
+      _ => {
+        entity(as[ReqPagedForgerStakesByForger]) { body =>
+          withNodeView { sidechainNodeView =>
+            val accountState = sidechainNodeView.getNodeState
+            val epochNumber = accountState.getConsensusEpochNumber.getOrElse(0)
+            if (Version1_4_0Fork.get(epochNumber).active) {
```

Comment:
Ok I removed the check here and added a TODO for checking the activation of the storage in the called method. It will be applied as soon as this branch is merged with the main code branch.